packages/snaps-controllers/src/snaps/SnapController.test.ts

```diff
@@ -4003,6 +4007,23 @@ describe('SnapController', () => {
       const oldVersion = '1.0.0';
       const newVersion = '1.0.1';
 
+      const manifest1 = (
+        await getMockSnapFilesWithUpdatedChecksum({
+          manifest: getSnapManifest({
+            version: newVersion,
+          }),
+        })
+      ).manifest.result;
+
+      const manifest2 = (
+        await getMockSnapFilesWithUpdatedChecksum({
+          manifest: getSnapManifest({
+            version: newVersion,
+          }),
+          sourceCode: 'foo',
+        })
+      ).manifest.result;
```

Comment:
Should one of these be using the old version?

Reply:
I don't believe so, since the old version is `1.0.0` and thus the default manifest.